.circleci/config.yml

```
@@ -125,27 +125,30 @@ jobs:
       - run:
           name: Qlik docker login
           command: echo $DOCKER_TOKEN | docker login -u "$DOCKER_USER" --password-stdin
+      - run:
+          name: Login to Github Project
+          command: echo "$APICULTURIST_TOKEN" | docker login ghcr.io -u "$APICULTURIST_GITHUB" --password-stdin
```

I think this should be
```suggestion
          command: echo "$APICULTURIST_TOKEN" | docker login ghcr.io/qlik-trial -u "$APICULTURIST_GITHUB" --password-stdin
```

---

Awesome, thanks @T-Wizard . I will give it a try